.github/build-iroha1.src.yml

```diff
@@ -0,0 +1,540 @@
+    # ## Check if PR same repo, so GitHub secrets are available, without them nothing to do
+    # if: ${{ github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name }}
+    runs-on: [ self-hosted, Linux ]
+    continue-on-error: true  ##FIXME  ##Sets needs.job_id.result always to success
+    steps:
+      - *step_show_context
+      - &step_cancel_previous_runs
+        if: ${{false}}
+        name: Cancel Previous Workflow Runs
+        uses: n1hility/cancel-previous-runs@v2
+        with:
+          token: ${{ github.token }}
+      - &step_system_info
+        name: System info
+        run: |
+          set -x
+          whoami
+          free  ||  vm_stat | perl -ne '/page size of (\d+)/ and $size=$1;
+            /Pages\s+([^:]+)[^\d]+(\d+)/ and printf("%-16s % 16.2f Mi\n", "$1:", $2 * $size / 1048576);'
+          df -h
+        #  ccache --show-stats
+      - &step_build_info
+        name: Build info
+        run: |
+          cat << 'END'
+          ref:${{github.ref}}
+          sha:${{github.sha}}
+          run_number:${{github.run_number}}
+          event_name:${{github.event_name}}
+          event.action:${{github.event.action}}
+          event.issue.number:${{ github.event.issue.number }}
+          END
+      - &step_detect_commented_pr
+        name: REF and SHA of commented PR to ENV
+        if: github.event.comment
+        run: >
+            curl -fsSL ${{github.event.issue.pull_request.url}}
+            -H "Authorization: token ${{github.token}}" |
+            jq -r '
+              "PR_REF="+.head.ref,
+              "PR_SHA="+.head.sha,
+              "PR_NUM="+(.number|tostring),
+              "PR_REPO="+.head.repo.full_name' >>$GITHUB_ENV
+      - &step_checkout
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref:        ${{env.PR_REF}}  ## not empty on issue_comment, else default value GITHUB_REF
+          repository: ${{env.PR_REPO}} ## not empty on issue_comment, else default value github.repository, required by forks
+      - &step_docker_tag
+        name: Determine dockertag
+        id: dockertag
+        env:
+          dockertag: ${{ hashFiles('docker/develop/Dockerfile.builder') }}
+        run: |
+          echo "::set-output name=dockertag::$dockertag"
+          echo >>$GITHUB_ENV  dockertag=$dockertag
+      - &step_docker_creds
+        ## This step is required since we cannot rely on secrets.DOCKERHUB_ORG
+        name: Dockerhub creds
+        env:
+          DOCKERHUB_ORG:      ${{ secrets.DOCKERHUB_ORG }}
+          DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+          DOCKERHUB_TOKEN:    ${{ secrets.DOCKERHUB_TOKEN }}
+          DOCKERHUB:          ${{ secrets.DOCKERHUB }}
+        run: |
+          DOCKERHUB_ORG=${DOCKERHUB_ORG:-hyperledger}
+          echo DOCKERHUB_ORG="$DOCKERHUB_ORG"                     >>$GITHUB_ENV
+          echo DOCKERHUB_USERNAME="${DOCKERHUB_USERNAME:-humb1t}" >>$GITHUB_ENV
```

Comment:
Humb1t is not maintainer anymore, so we should change secrets.

Reply:
Very good point. Thank you.

Reply:
removed